docs/content/users/topics/sharing.md

```diff
@@ -46,6 +46,14 @@ This set of steps assumes an ngrok subdomain `mg2`:
 * Run `bin/magento setup:store-config:set --base-url="https://mg2.ngrok.io/`.
 * Run [`ddev share`](../usage/commands.md#share) and you’ll see your project at `mg2.ngrok.io`.
 
+## Using localhost.run service
```

Comment:
I’d drop “service” here, or if you feel strongly about keeping it capitalize the “S” so the heading is title cased:

```suggestion
## Using localhost.run
```

Reply:
Hi @mattstein  Thanks for the review. I have kept service by making S capital. For better understanding.